cypress/integration/models/administrator/general/generalConfig.ts

```diff
@@ -4,55 +4,65 @@ import {
     enableSwitch,
     selectUserPerspective,
 } from "../../../../utils/utils";
-import { downloadCSV, downloadHTML } from "../../../views/reports.view";
+import { downloadCSV, downloadHTML, reviewAssessment } from "../../../views/reports.view";
 import { administration, general } from "../../../types/constants";
 import { navMenu } from "../../../views/menu.view";
 
-export class ReportConfig {
-    private static instance: ReportConfig;
+export class GeneralConfig {
+    private static instance: GeneralConfig;
     static downloadHtml: boolean;
     static downloadCsv: boolean;
     static fullUrl = Cypress.env("tackleUrl") + "/general";
 
     // ReportConfig class is singleton, which means that only one object of this class can be created
     // This function is required to get ReportConfig instance in any part of the code
-    public static getInstance(): ReportConfig {
-        if (!ReportConfig.instance) {
-            ReportConfig.instance = new ReportConfig();
+    public static getInstance(): GeneralConfig {
+        if (!GeneralConfig.instance) {
+            GeneralConfig.instance = new GeneralConfig();
         }
-        return ReportConfig.instance;
+        return GeneralConfig.instance;
     }
 
     static openConfig(): void {
         cy.url().then(($url) => {
-            if ($url != ReportConfig.fullUrl) {
+            if ($url != GeneralConfig.fullUrl) {
                 selectUserPerspective(administration);
                 clickByText(navMenu, general);
             }
         });
     }
 
     enableDownloadHtml(): void {
-        ReportConfig.openConfig();
+        GeneralConfig.openConfig();
         enableSwitch(downloadHTML);
-        ReportConfig.downloadHtml = true;
+        GeneralConfig.downloadHtml = true;
```

Comment:
generalConfig (g lower case)

Reply:
It is a static property it, should be in upper case because it is calling the class directly.

Reply:
Hi @kpunwatk @abrugaro Yes for class name Upper case , for static variable I couldn't find any well written naming convention but for other programming languages it is . So let's keep it upper case.

Reply:
TS classes should start in uppercase and properties (static or not) in snakeCase. So `GeneralConfig.downloadHTML` was correct

Reply:
Done. Revert backed the changes